app/assets/stylesheets/spotlight/_featured_browse_categories_block.scss

```diff
@@ -0,0 +1,69 @@
+.browse-category {
```

Comment:
This is more a note for later, but we should use bootstrap variables (or calculations based on those variables..) for the magic numbers below.

I also vaguely wonder if this is another use case for `flexbox`.